### PR TITLE
Refactor the frontend

### PR DESCRIPTION
**Pull request by DeKleineKobini**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -5,7 +5,8 @@
     </li>
     <li class="nav-item">
       <a *ngIf="!isLoggedIn()" routerLink="login" routerLinkActive="active" class="nav-link">Log in now</a>
-      <button (click)="logout()" *ngIf="isLoggedIn()" routerLink="" routerLinkActive="active" class="nav-link">Log out</button>
+      <button (click)="logout()" *ngIf="isLoggedIn()" routerLink="" routerLinkActive="active" class="nav-link">Log out
+      </button>
```
Comment (AhmadAlhajKarim) [inline src/app/app.component.html]: Maybe some formatting?

File: src/app/components/login/login.component.ts [hunk 2]
```diff
@@ -17,27 +17,31 @@ export class LoginComponent implements OnInit {
...
+        .subscribe({
+          complete: () => {
+            // noinspection JSIgnoredPromiseFromCall
+            this.router.navigate(['']);
+            this.badCredentials = false;
+          },
+          error: () => this.badCredentials = true,
+        });
     } else {
       // this.empty= true;
```
Comment (TomD011099) [inline src/app/components/login/login.component.ts]: Empty else?

File: src/index.html [hunk 3]
```diff
@@ -6,9 +6,9 @@
   <base href="/">
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" />
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css"/>
```
Comment (AhmadAlhajKarim) [inline src/index.html]: Bootstrap is added normally in angular.json and not needed to be here i guess

**Review by AhmadAlhajKarim — CHANGES_REQUESTED** [review]

File: src/app/components/login/login.component.ts (hunk 2 above)
Comment (DeKleineKobini) [reply src/app/components/login/login.component.ts]: fixed

**Review by TomD011099 — APPROVED** [review]

**Review by AhmadAlhajKarim — APPROVED** [review]

**Review by AhmadAlhajKarim — APPROVED** [review]

**Review by AhmadAlhajKarim — APPROVED** [review]

**Review by AhmadAlhajKarim — APPROVED** [review]

**Pull request merged**